### PR TITLE
Fixed export VCENTER_USERNAME in GSG & generate-yaml.sh script

### PR DESCRIPTION
**Pull request by tkrausjr**
**What this PR does / why we need it**:
Simple fix for `export VSPHERE_USER=viadmin@vmware.local `     changed to  `export VSPHERE_USERNAME=viadmin@vmware.local `

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
 Isssue #433 

**Special notes for your reviewer**:
Document Change to https://github.com/tkrausjr/cluster-api-provider-vsphere/blob/master/docs/getting_started.md
_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by andrewsykim** [comment]
Thanks for the PR @tkrausjr! Can you also update `VSPHERE_USER` [here](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/67e7ef36267ff66a42bfaa6a1198286d9922164c/hack/generate-yaml.sh#L162)?

**Comment by tkrausjr** [comment]
LOL @andrewsykim   I just noticed VSPHERE_USER in generate-yaml.sh myself and was coming in to fix this. Any idea the best way to do that from a git standpoint ? Do I close this and create a new PR ?

**Comment by andrewsykim** [comment]
If you push a new commit on your existing branch it should update this PR. Ping me on Kubernetes slack if you need a hand (@a_sykim)!

**Comment by tkrausjr** [comment]
Sorry yeah just figured that out. Its done.

**Comment by andrewsykim** [comment]
Awesome, make sure you sign the CLA (see comment https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/434#issuecomment-511559474)

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (andrewsykim) [inline .vscode/settings.json]: remove this file please :)

Comment (tkrausjr) [reply .vscode/settings.json]: Sorry done.

**Comment by figo** [comment]
@tkrausjr thanks for the PR, please squash the 3 commits into one.

**Comment by tkrausjr** [comment]
@andrewsykim  I signed it a while ago as part of my corporation I just need to change my email address associated with my github account which is the problem. Do you need this fixed before merge ?

**Comment by andrewsykim** [comment]
@tkrausjr unfortunately yes! CLA is a requirement for kubernetes repos.

**Comment by tkrausjr** [comment]
"I signed it"

**Comment by tkrausjr** [comment]
@figo  I squashed 3 into 1.

File: docs/getting_started.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ The project Makefile provides a convenient make target to generate the yaml file
 ```bash
 # vCenter config/credentials
 export VSPHERE_SERVER=10.0.0.1                # (required) The vCenter server IP or UR
-export VSPHERE_USER=viadmin@vmware.local      # (required) The vCenter user to login with
+export VSPHERE_USERNAME=viadmin@vmware.local      # (required) The vCenter user to login with
```
Comment (akutz) [inline docs/getting_started.md]: Hi @tkrausjr,

The alignment on line 99 is now incorrect. Please fix or let this PR be replaced by #412. Either way is fine.

Comment (tkrausjr) [reply docs/getting_started.md]: Done

**Comment by andrewsykim** [comment]
/approve
/lgtm 

Thank you @tkrausjr!

**Comment by tkrausjr** [comment]
@akutz  We made the same change to the GSG, This PR also fixes hack/generate-yaml.sh ENV Check for VSPHERE_USERNAME.

I dont know how to fix this conflict can you ?

**Comment by akutz** [comment]
Hi @tkrausjr,

I can help you fix it, yes. Assuming you're on your branch:

```shell
{ git remote add upstream https://github.com/kubernetes-sigs/cluster-api-provider-vsphere || true; } && \
git fetch upstream && \
git checkout upstream/master -- docs/getting_started.md && \
{ git add docs/getting_started.md || true; } && \
git commit --amend -C HEAD && \
git push --force
```

**Pull request merged**